pkg/config/config.go

```diff
@@ -146,15 +146,15 @@ func NewConfig(confString string, c *cli.Context) (*Config, error) {
 			ActiveLevel:     30, // -30dBov = 0.03
 			MinPercentile:   40,
 			UpdateInterval:  500,
-			SmoothIntervals: 4,
+			SmoothIntervals: 2,
```

Comment:
Test found that SmoothInterval set to 4 willl cause the speaker state reaction  a little slow. I have test set default to 2 will more sensitive. But don't know is there other reason to set SmoothIntervals to 4, if it has, can revert this change